ui/tournament/src/view/playerInfo.ts

```diff
@@ -104,8 +104,14 @@ export default function (ctrl: TournamentController): VNode {
                 h('th', '' + (Math.max(nb.game, pairingsLen) - i)),
                 h('td', playerName(p.op)),
                 ctrl.opts.showRatings ? h('td', p.op.rating) : null,
+                p.berserk
+                  ? h('td.berserk', { attrs: { 'data-icon': licon.Berserk, title: 'Berserk' } })
+                  : h('td.berserk'),
                 h('td.is.color-icon.' + p.color),
-                h('td', res),
+                h('td' + '.result.', res),
+                p.berserk
```

Comment:
This is displaying the same info twice: `p.berserk`

Reply:
🤦. I've got no idea how I didn't notice that